content/azure-managed-identity.md

```diff
@@ -0,0 +1,9 @@
+You can configure BOSH to use [Azure Managed Identities](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm) to avoid hard coding specific Azure credentials.
+Azure Managed Identities are similar to AWS instance profiles.  
+You first have to create an Azure Managed Identity, and give the proper roles/permissions required by BOSH (i.e. create/delete VM, create/delete/attach disks).
+
+Theres is an [azure/use-managed-identity.yml operator file](https://github.com/cloudfoundry/bosh-deployment/blob/master/azure/use-managed-identity.yml) that you can use.
```

Comment:
To stay in line with the aws-iam-instance-profile docs it would be better to describe which properties to set and how they work. That way the docs don't require people to understand ops-files. Ops-files are just a templating solution which some people use, however there other yaml templating tools out there used by people to deploy bosh ([spruce](https://github.com/geofffranks/spruce), [spiff++](https://github.com/mandelsoft/spiff), etc).